app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::UsersController < ApplicationController
+  def create
+    new_user = User.new(user_params)
+    if new_user.save
+      render json: UsersSerializer.new(new_user)
+    else
+      #400 error with description of why the request failed
```

Comment:
Planning to resume tomorrow with sad path tests and learning how to return json 400 error